Makefile

```diff
@@ -40,7 +40,7 @@ else
 endif
 
 .PHONY: test
-test: build-tests run-tests
+test: clean build-tests run-tests
```

Comment:
As our tests don't create any volumes, we shouldn't need to run the clean target as well.